### PR TITLE
Add RandomBeaconHistory to core contracts

### LINKED ISSUE #499: Add `RandomBeaconHistory` to Core Contracts
`RandomBeaconHistory` is now a core contract and needs to be added: https://github.com/onflow/flow-emulator/blob/0cdc31ceecde40ee7ffbf70b8e77961b7bbd7ce4/server/server.go#L166

### PR DESCRIPTION
**Pull request by chasefleming**
Closes #499 

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to GitHub issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Comment by bluesign** [comment]
I think this should come from flow-go, not sure about the required update there. cc: @janezpodhostnik

File: server/server.go [hunk 1]
```diff
@@ -164,14 +164,15 @@ func NewEmulatorServer(logger *zerolog.Logger, conf *Config) *EmulatorServer {
...
-		"MetadataViews":      chain.ServiceAddress().HexWithPrefix(),
+		"FlowServiceAccount":  chain.ServiceAddress().HexWithPrefix(),
+		"FlowToken":           fvm.FlowTokenAddress(chain).HexWithPrefix(),
+		"FungibleToken":       fvm.FungibleTokenAddress(chain).HexWithPrefix(),
+		"FlowFees":            environment.FlowFeesAddress(chain).HexWithPrefix(),
+		"FlowStorageFees":     chain.ServiceAddress().HexWithPrefix(),
+		"NonFungibleToken":    chain.ServiceAddress().HexWithPrefix(),
+		"ViewResolver":        chain.ServiceAddress().HexWithPrefix(),
+		"MetadataViews":       chain.ServiceAddress().HexWithPrefix(),
+		"RandomBeaconHistory": chain.ServiceAddress().HexWithPrefix(),
```
Comment (tarakby) [inline server/server.go:175]: @janezpodhostnik should know better

**Review by janezpodhostnik — APPROVED** [review]

**Comment by janezpodhostnik** [comment]
This is only for logging which core contracts are deployed where, to inform the user. 

Ideally this would either be something that we can get from flow-go, or queried using a script.  For now this update is ok.

**Review by bluesign — APPROVED** [review]

**Comment by chasefleming** [comment]
@janezpodhostnik you said this is deployed to the service account though like the other contracts already, is that correct?

**Review by sideninja — APPROVED** [review]

**Comment by chasefleming** [comment]
Confirmed it's on the service account. Just need to log it.

**Pull request merged**